packages/neuron-wallet/src/models/chain/witness-args.ts

```diff
@@ -0,0 +1,45 @@
+export default class WitnessArgs {
+  public static EMPTY_LOCK = '0x' + '0'.repeat(130)
+
+  constructor(
+    public lock?: string,
+    public inputType?: string,
+    public outputType?: string
+  ) {
+    this.outputType = outputType
```

Comment:
Why do we need to assign this property manually?

Reply:
I forgot to delete this line too.